astroid/brain/brain_namedtuple_enum.py

```diff
@@ -379,7 +379,7 @@ def infer_enum_class(node: nodes.ClassDef) -> nodes.ClassDef:
                 continue
 
             inferred_return_value = None
-            if isinstance(stmt, (nodes.AnnAssign, nodes.Assign)):
+            if isinstance(stmt, (nodes.AnnAssign, nodes.Assign)) and stmt.value:
```

Comment:
`stmt` will always be of type `AnnAssign` or `Assign`. If it isn't, `continue` is triggered a few lines above. Thus, only checking for `stmt.value` is enough.
https://github.com/PyCQA/astroid/blob/492a4bfde15fdd4e14de7a93a160a9fafa6ecf46/astroid/brain/brain_namedtuple_enum.py#L371-L382
```suggestion
            if stmt.value is not None:
```

Reply:
Hats off that makes perfect sense!